transforms/code/repo_level_ordering/ray/src/dpk_repo_level_order/internal/repo_level_wrappers.py

```diff
@@ -12,24 +12,33 @@
     sort_by_path,
     sort_sem,
 )
+from func_timeout.exceptions import FunctionTimedOut
 
 
 SORT_BY_PATH = "SORT_BY_PATH"
 SORT_SEMANTIC = "SORT_SEMANTIC"
 SORT_SEMANTIC_NORMALISED = "SORT_SEMANTIC_NORMALISED"
 
 
-def semantic_sort(df, logger, title_column_name):
-    return sort_sem(files_df=df, logger=logger, title_column_name=title_column_name)
+def semantic_sort(df, logger, title_column_name, language_column_name):
+    return sort_sem(
+        files_df=df, logger=logger, title_column_name=title_column_name, language_column_name=language_column_name
+    )
 
 
-def semantic_sort_normalised(df, logger, title_column_name):
+def semantic_sort_normalised(df, logger, title_column_name, language_column_name):
```

Comment:
instead of having two functions(`semantic_sort` and `semantic_sort_normalised`, can we combine them into one by adding one additional optional parameter "normalize_title" with default value of `False`

```
semantic_sort(df, logger, title_column_name, language_column_name, normalize_title=False):
```